gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineIdeaIntegrationTest.groovy

```diff
@@ -70,36 +70,6 @@ class BaselineIdeaIntegrationTest extends AbstractPluginTest {
         with('idea').build()
     }
 
-    def 'Modules for subprojects pick up the correct sourceCompatibility'() {
-        when:
-        buildFile << standardBuildFile
-        buildFile << '''
-            sourceCompatibility = 1.6
-        '''.stripIndent()
-        def subproject = multiProject.create(["subproject1", "subproject2"])
-        subproject["subproject1"].buildGradle << '''
-            apply plugin: 'java'
-            apply plugin: 'com.palantir.baseline-idea'
-            sourceCompatibility = 1.7
-        '''.stripIndent()
-        subproject["subproject2"].buildGradle << '''
-            apply plugin: 'java'
-            apply plugin: 'com.palantir.baseline-idea'
-            sourceCompatibility = 1.8
```

Comment:
after this change, what's the resulting config in the iml xml?

Reply:
I ran it on our internal foundry email-service and it gave me this `idea.iml` file:
```
<?xml version="1.0" encoding="UTF-8"?>
<module relativePaths="true" type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager" inherit-compiler-output="true">
    <exclude-output />
    <content url="file://$MODULE_DIR$">
      <excludeFolder url="file://$MODULE_DIR$/.gradle" />
      <excludeFolder url="file://$MODULE_DIR$/build" />
    </content>
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
  </component>
</module>
```